src/directives/components/graphdb/graphdb.directive.html

```diff
@@ -0,0 +1,95 @@
+    </property-html>
+
+    <!-- RDF Rank -->
+    <property-html ng-if="graphdb.rdfRankStatus.indexOf('COMPUTED') === 0"
+                   label-key="tt2ps.components.graphdb.property.rdfRankStatus.label"
+                   html="true"
+                   value="{{'<span class=\'tag tag-success\'><i class=\'rdf-rank-status-icon fa fa-lg fa-circle-check\'></i>' + ('tt2ps.components.graphdb.property.rdfRankStatus.computed' | translate) + '</span>'}}">
+    </property-html>
+
+    <property-html ng-if="graphdb.rdfRankStatus.indexOf('ERROR') === 0"
+                   label-key="tt2ps.components.graphdb.property.rdfRankStatus.label"
+                   html="true"
+                   value="{{'<span class=\'tag tag-danger\'><i class=\'rdf-rank-status-icon fa fa-lg fa-warning\'></i>' + graphdb.rdfRankStatus + '</span>'}}">
+    </property-html>
+
+    <property-html ng-if="graphdb.rdfRankStatus === 'CANCELED'"
+                   label-key="tt2ps.components.graphdb.property.rdfRankStatus.label"
+                   html="true"
+                   value="{{'<span class=\'tag tag-warning\'><i class=\'rdf-rank-status-icon fa fa-lg fa-warning\'></i>' + ('tt2ps.components.graphdb.property.rdfRankStatus.canceled' | translate) + '</span>'}}">
+    </property-html>
+
+    <property-html ng-if="graphdb.rdfRankStatus === 'CONFIG_CHANGED'"
+                   label-key="tt2ps.components.graphdb.property.rdfRankStatus.label"
+                   html="true"
+                   value="{{'<span class=\'tag tag-warning\'><i class=\'rdf-rank-status-icon fa fa-lg fa-warning\'></i>' + ('tt2ps.components.graphdb.property.rdfRankStatus.config-changed' | translate) + '</span>'}}">
+    </property-html>
+
+    <property-html ng-if="graphdb.rdfRankStatus === 'OUTDATED'"
+                   label-key="tt2ps.components.graphdb.property.rdfRankStatus.label"
+                   html="true"
+                   value="{{'<span class=\'tag tag-warning\'><i class=\'rdf-rank-status-icon fa fa-lg fa-warning\'></i>' + ('tt2ps.components.graphdb.property.rdfRankStatus.outdated' | translate) + '</span>'}}">
+    </property-html>
+
+    <property-html ng-if="graphdb.rdfRankStatus === 'COMPUTING'"
+                   label-key="tt2ps.components.graphdb.property.rdfRankStatus.label"
+                   html="true"
+                   value="{{'<span class=\'tag tag-info\'><i class=\'rdf-rank-status-icon fa fa-lg fa-refresh fa-spin\'></i>' + ('tt2ps.components.graphdb.property.rdfRankStatus.computing' | translate) + '</span>'}}">
+    </property-html>
+
+    <property-html ng-if="graphdb.rdfRankStatus === 'EMPTY'"
+                   label-key="tt2ps.components.graphdb.property.rdfRankStatus.label"
+                   html="true"
+                   value="{{'<span class=\'tag tag-warning\'><i class=\'rdf-rank-status-icon fa fa-lg fa-info-circle\'></i>' + ('tt2ps.components.graphdb.property.rdfRankStatus.empty' | translate) + '</span>'}}">
```

Comment:
I think CANCELED and EMPTY should be with tag-danger

Reply:
I’ll update them according to your suggestion.